open-vm-tools/vmblockmounter/vmblockmounter.c

```diff
@@ -147,7 +154,7 @@ main(int argc,          // IN
       thisProgramBase = thisProgram;
    }
 
-   while ((c = getopt(argc, argv, "hvV")) != -1) {
+   while ((c = getopt(argc, argv, "hvV")) != GETOPT_END) {
```

Comment:
can you define c as int instead?

Reply:
I had assumed that getopt return was of type "char" since it was dealing with character data. But looking further into it, both Linux and FreeBSD define it as "int", so yup, that would be the better code path here for language/OS compatibility.